docs/integrations-guides/rollup-guides/integrations-overview.md

```diff
@@ -77,70 +87,64 @@ is disabled, and state roots cannot be challenged. This means the sequencer can
 post whatever state roots they want to the bridge contract and potentially steal
 funds.
 
-## L2 Inbox Certificate Verification Strategy (M1) {#M1}
+## Cert Timeliness Verification
+
+EigenDA blobs are only available to download for 2 weeks, so it is important
+to ensure that the [batcher][glossary-batcher] is not posting EigenDA certs to the rollup inbox after the blob has been deleted. Each securely integrated rollup stack should have a [cert-punctuality-window][glossary-cert-punctuality-window] defined by its derivation pipeline.
+
+## Cert Verification
+
+Cert validity rules are encoded in the EigenDACertVerifier contract. Cert validity can thus be checked
+offchain by making an eth-call, or onchain by calling the respective method. It can also be zk proven via a storage proof. See our [V2 integration spec][spec-cert-validation]. Ultimately though, the L1 chain must be
+convinced that the cert is valid, which can either be done:
+1. Pessimistically
+   1. verify in the [rollup-inbox][glossary-rollup-inbox] contract for every blob (optimistic rollups)
+   2. create a zk proof which is aggregated and submitted along with the state transition correctness proof (zk rollups)
+2. Optimistically: only verify during one step proving if/when a fraud happens (optimistic rollups)
+
+Although the pessimistic implementation is simpler, the optimistic approach is
+often desirable since verification only incurs on-chain costs when the sequencer is
+dishonest.
+
+### Pessimistic Cert Verification
+
+We only describe the inbox verification strategy here as it is mostly straightforward. There are many different ways to get a zk proof of storage, so teams wanting to use this approach should consult their relevant stack's guide.
 
-The natural locations for dispersal, retrieval, and data verification logic are
-unambiguous – dispersal logic can only live inside the rollup batcher, retrieval
-logic can only live inside the derivation pipeline, and data verification logic
-can only live inside the fraud proof system. That leaves certificate
-verification logic, which could be placed either in the rollup's inbox contract
-or inside the rollup virtual machine executing off-chain. The question of where
-to place this logic is the key distinguishing factor between the "M1" and
-"M2" integration paths.
+> Note: this strategy is only possible for rollup stacks whose [rollup-inbox][glossary-rollup-inbox] is a contract (e.g. arbitrum nitro). On the op stack, the batcher inbox is an EOA so it is not possible for it to make calls to the DACertVerifier (lest it uses [eip-7702](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7702.md)).
```

Comment:
> lest it uses

I'm not sure what you mean with "lest" here. Did you mean to say "since"?

Reply:
heh I don't know english, pardon me. Meant "unless"

Reply:
[c3300f7](https://github.com/Layr-Labs/eigenda-docs/pull/72/commits/c3300f7022b2859f7bc690eae380514975f4162a)